### PR TITLE
#trivial Rewrite Spinner component to typescript

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
As the title says.
#1084 

#### How should this be manually tested?
`yarn && yarn start`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Spinner/index.tsx [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
-        height={size}
-        width={size}
-      >
-        <style>
-          {`
+const Spinner: FC<Props> = ({ color, size, block }: Props) => (
+  <svg
+    className={`${baseClassname('spinner')} ${
+      !color ? 'c-action-primary' : ''
+    } ${block ? 'db' : ''}`}
```
Comment (emersonlaurentino) [inline react/components/Spinner/index.tsx:24]: can u use classNames lib?

Comment (thayannevls) [reply react/components/Spinner/index.tsx]: Done

File: react/components/Spinner/index.tsx [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
-        } ${block ? 'db' : ''}`}
-        xmlns="http://www.w3.org/2000/svg"
-        viewBox="0 0 100 100"
-        preserveAspectRatio="xMidYMid"
-        height={size}
-        width={size}
-      >
-        <style>
-          {`
+const Spinner: FC<Props> = ({ color, size, block }: Props) => (
```
Comment (JuanBarros2) [inline react/components/Spinner/index.tsx:21]: nit: I think that you could omit the props definition in brackets `const Spinner: FC<Props> = ({ color, size, block }) `

Comment (thayannevls) [reply react/components/Spinner/index.tsx:21]: Oops, sorry :sweat_smile: This was something that I was testing

Comment (thayannevls) [reply react/components/Spinner/index.tsx]: Resolved

File: react/components/Spinner/index.tsx [hunk 3]
```diff
@@ -1,28 +1,37 @@
...
-        viewBox="0 0 100 100"
-        preserveAspectRatio="xMidYMid"
-        height={size}
-        width={size}
-      >
-        <style>
-          {`
+const Spinner: FC<Props> = ({ color, size, block }) => (
+  <svg
+    className={classNames(`${baseClassname('spinner')}`, {
```
Comment (emersonlaurentino) [inline react/components/Spinner/index.tsx:23]:
```suggestion
    className={classNames(baseClassname('spinner'), {
```

Comment (thayannevls) [reply react/components/Spinner/index.tsx]: Done

**Pull request merged**